tractor/_spawn.py

```diff
@@ -501,6 +523,11 @@ async def mp_new_proc(
         # local actor by the time we get a ref to it
         event, chan = await actor_nursery._actor.wait_for_peer(
             subactor.uid)
+
+        # XXX: monkey patch poll API to match the ``subprocess`` API..
+        # not sure why they don't expose this but kk.
+        proc.poll = proc._popen.poll  # type: ignore
```

Comment:
@richardsheridan mentioned on matrix that `proc.exitcode` is actually already this + some error checking - so maybe use that?

Reply:
Oh right, but i guess then we have to wrap reading that attr as an equivalent duck-typed `.poll()`?